deploy/rest/config.yaml

```diff
@@ -0,0 +1,231 @@
+      query:
+        type: code
+      documents:
+        type: json
+        default: []
+  code:
+    Code:
+      _path: superduperdb/Code
+      identifier:
+        type: str
+      code:
+        type: str
+        default: |
+          from superduperdb import code
+
+          @code
+          def my_code(x):
+            return x
+  lazy_artifact:
+    LazyArtifact:
+      _path: superduperdb/components/datatype/LazyArtifact
+      identifier:
+        type: str
+      file_id:
+        type: blob
+  vector_index:
+    VectorIndex:
+      _path: superduperdb/VectorIndex
+      identifier:
+        type: str
+      measure:
+        type: str
+        choices:
+          - cosine
+          - dot
+          - l2
+      indexing_listener:
+        type: listener
+      compatible_listener:
+        type: listener
+        optional: True
+  datatype:
+    image:
+      _path: superduperdb/ext/pillow/image_type
+      identifier:
+        type: str
+      media_type:
+        type: str
+        default: image/png
+    vector:
+      _path: superduperdb/vector
+      identifier:
+        type: str
+      shape:
+        type: int
+  stack:
+    Stack:
+      _path: superduperdb/Stack
+      identifier:
+        type: str
+      components:
+        type: [model, listener, vector_index]
+  listener:
+    Listener:
+      _path: superduperdb/Listener
+      identifier:
+        type: str
+      key:
+        type: str
+      select:
+        type: query
+        optional: True
+  model:
+    ObjectModel: 
+      _path: superduperdb/ObjectModel
+      identifier:
+        type: str
+      object:
+        type: lazy_artifact
+      datatype:
+        type: datatype
+        optional: True
+      predict_kwargs:
+        type: json
+        optional: True
+        default: {}
+      signature:
+        type: str
+        optional: True
+        default: "*args,**kwargs"
+    SequentialModel:
+      _path: superduperdb/SequentialModel
+      identifier:
+        type: str
+      models:
+        type: model
+        sequence: True
+    QueryModel:
+      _path: superduperdb/QueryModel
+      identifier:
+        type: str
+      select:
+        type: query
+        optional: True
+        default:
+          documents:
+          - {"<key-1>": "$my_value"}
+          - {"_outputs": 0, "_id": 0}
+          query: | 
+            <collection_name>.like(documents[0], vector_index='<index_id>').find({}, documents[1]).limit(10)
+    CodeModel:
+      _path: superduperdb/CodeModel
+      identifier:
+        type: str
+      object:
+        type: code
+      datatype:
+        type: datatype
+        optional: True
+      predict_kwargs:
+        type: json
+        optional: True
+        default: {}
+      signature:
+        type: str
+        optional: True
+        default: "*args,**kwargs"
+    RetrievalPrompt: 
+      _path: superduperdb/ext/llm/prompt/RetrievalPrompt
+      select:
```

Comment:
Here. I think the select field should be enclosed. Otherwise frontend can't render forms.
"select": {
        "type": "query",
}